### PR TITLE
feat (examples): Added Swift examples

### PR DESCRIPTION
**Pull request by craigtweedy**
# Description

Adds two example Swift projects which show how to use GraphCMS with Swift. One is native and uses `URLSession` and plain GraphQL strings, while the other uses [swift-graphql](https://github.com/maticzav/swift-graphql) to parse a GraphQL schema into native Swift and build queries from that.

Both examples are built in SwiftUI, with (rough) support for varying devices (iPhone / iPad / Mac). Both use Swift's native `async/await` support.

Swift-GraphQL-Phone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,8 @@ frameworks.
 | [Astro] ([Demo][demo29])                                              | An Astro example using `graphql-request` to fetch data.                                                                  |
 | [Vanilla JS] ([Demo][demo30])                                         | Query in the browser with no frameworks, just use the Fetch API.                                                         |
 | [NextAuth] ([Demo][demo31])                                           | Authenticate with NextAuth.js and update account info with GraphCMS.                                                     |
+| [Swift] ([Demo](with-swift))                                                               | A native Swift (iOS & Mac) example.                                                     |
```
Comment (notrab) [inline README.md:125]: Typically the `demo` is something the user can click on to try out straight away. Is there a hosted service that we could use for something similar?

File: README.md [hunk 2]
```diff
@@ -122,6 +122,8 @@ frameworks.
...
+| [Swift with `swift-graphql`] ([Demo][swift-graphql-demo])                                                              | A native Swift (iOS & Mac) example using [swift-graphql](swift-graphql).                                                     |
```
Comment (notrab) [inline README.md:126]: Typically the `demo` is something the user can click on to try out straight away. Is there a hosted service that we could use for something similar?

File: with-swift-graphql/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# GraphCMS x Swift with swift-graphql
+
+[Join our Slack](https://slack.graphcms.com)
+
+This demonstrates how to use GraphCMS with a Swift application for iOS (iPhone / iPad) or Mac, using popular library [swift-graphql](https://github.com/maticzav/swift-graphql)
```
Comment (notrab) [inline with-swift-graphql/README.md:5]: Were all issues resolved you ran into @craigtweedy? Maybe worth making a note below if there are any caveats.

File: README.md (hunk 2 above)
Comment (craigtweedy) [reply README.md]: Good spot. Was some overzealous copy/pasting.

File: with-swift-graphql/README.md (hunk 3 above)
Comment (craigtweedy) [reply with-swift-graphql/README.md:5]: With regards to the schema generation? They weren't. 

I can make a note about it - will need to look into whether it's a problem with just this schema or wider. I'll have to raise the core issues with the maintainer as it's a good test bed anyway.

**Review by notrab — APPROVED** [review]

**Comment by notrab** [comment]
@craigtweedy I'm happy to merge if you are.

**Pull request merged**